docs/pages/blog/2023-toolpad-beta-announcement.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Toolpad is now in Beta!
+description: After working on it for more than a year, Toolpad is now in the next stage. Read more about it in this announcement.
+date: 2023-07-03T00:00:00.000Z
+authors: ['prakhargupta']
+card: true
+tags: ['Product', 'News']
+---
+
+It's been over a year since we released the first version of Toolpad. Today, we're excited to take the next step on that journey with the release of Toolpad Beta. If you aren't familiar with Toolpad yet, it's an admin panel builder catering to the internal tooling needs of an organization. It's for developers who want to build a functional application quickly. It harnesses the speed of a UI builder for the front-end and closely integrates with your normal development workflow. If this excites you, then read on!
```

Comment:
Does it make sense to put backend vs. frontend here

```suggestion
It's been over a year since we released the first version of Toolpad. Today, we're excited to take the next step on that journey with the release of Toolpad Beta. If you aren't familiar with Toolpad yet, it's an admin panel builder catering to the internal tooling needs of an organization. It's for developers who want to build a functional application quickly. It harnesses the speed of a UI builder for the frontend and closely integrates into your backend. If this excites you, then read on!
```

Reply:
Makes sense.

I used 'front-end' and not front end or frontend form: https://mui.com/material-ui/getting-started/installation/#cdn
and that led to 'back-end' in this entire post.